text/0000-read-only-build-layers.md

```diff
@@ -59,6 +59,8 @@ Updates to the lifecycle so that `build` layers created by buildpacks should be
 
 The lifecycle will have to layerize the layers marked `build` as `true` during the `build` phase of the lifecycle instead of the `export` phase.
```

Comment:
Should all the necessary layers (anything `cache=true` or `launch=true`) during build phase regardless of whether they are `build=true`? This might be good for two reasons
1. to prevent changes to launch layers from downstream buildpacks (although this is admittedly harder to do accidentally)
2. for consistency. It might be weird to layerize things in two different phases